lib/schemas/config.js

```diff
@@ -9,7 +9,7 @@ module.exports = joi.object().keys({
     freezeConfig: joi.bool()
   }).required().unknown(),
 
-  env: joi.string().required(),
+  env: joi.string(), //.required(), // scott-wyatt: env is often undefined and can not be required.
```

Comment:
Why ? We should be default put `development` when it's undefined so I'll would let it required

Reply:
I don't think it should, I think that we can use undefined to target a local environment in the future.

Reply:
NODE_ENV is a common but not complete node protocol, so it's reasonable that it should be undefined.

Reply:
Using undefined as "value" doesn't look like a good idea to me, if we want a local environment it's better to create one by setting a custom value under env. For example my local env is set to development but if I want a different one I'll create another entry into env.
Also by setting it required trailpack can rely on it to know the env if needed, if it can be undefined then they need to add extra logic to handle the case...

Reply:
`undefined` in this case just means that trails doesn't need to go searching in `config/env` all trailpacks can assume that an undefined node_env means they should go to default which could equal `development`, but why define a value if it doesn't need to be?

Reply:
Just to be consistent and offer some stability/tranquility for trailpacks, if it's undefined and should set there own default it can become inconsistent as trailpack 1 can sa it's development and another one testing, by centralizing it here I find this more clear safe. But anyway not a big deal even if it can become undefined :)